Dockerfile

```diff
@@ -3,8 +3,7 @@
 # Initially based upon:
 # https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 
-FROM node:12.18.3-buster-slim@sha256:dd6aa3ed10af4374b88f8a6624aeee7522772bb08e8dd5e917ff729d1d3c3a4f
-    
+FROM node:14.15.4-buster-slim@sha256:8e4843f26333ed01f8733c7163bf88fc31c8516f96c46094d15e906160451f26
```

Comment:
Just curious: why is the commit hash needed? — I mean, provided the tag won't get updated and so on.

Reply:
Not sure. I just followed same approach as in original file. :)